libs/ngx-cron-editor/src/cron-editor.component.ts

```diff
@@ -213,21 +213,27 @@ export class CronGenComponent implements OnInit, OnDestroy, ControlValueAccessor
         break;
```

Comment:
Although the `tabIndex` is set to 0 initially, this might require setting the `tabIndex` back to 0 when the cron expression changes back to a minutely value.
If the component is not reloaded, then I believe it would get stuck on another tab if the value were to become minutely.

The usage of selecting the tab by its index I believe would break when hiding tabs.
*ngif removes tabs from the dom, so the index for it is not actually used up when rendering the tabs.

i.e. 
The Minutes Tab is in position 0 if it were rendered.
If `options.hideMinutesTab` is true, then index 0 would actually be the Hourly Tab, so the switch statement would select the wrong index when setting the expression.

Reply:
How to select a tab by its label:

```
@ViewChild('tabGroup') tabGroup! : MatTabGroup;

ngAfterViewInit() {
  this.setTabByLabel();
}

setTabByLabel() {
  let tab = this.tabGroup._allTabs.toArray().find((tabItem: MatTab) => tabItem.textLabel === this.tabName);
  if (tab && !tab.disabled) {
    this.tabGroup.selectedIndex = this.tabGroup._allTabs.toArray().indexOf(tab);
  }
}
```